pysep/app.json

```diff
@@ -0,0 +1,40 @@
+{
+  "id": "pysep",
+  "owner": "${apiUserId}",
+  "version": "1.0",
+  "description": "Containerized pysep jupyter nb",
+  "jobType": "BATCH",
+  "runtime": "SINGULARITY",
+  "runtimeOptions": ["SINGULARITY_RUN"],
+  "containerImage": "docker://jthet/pysep:latest",
```

Comment:
I think we should change this to have the value of the official scoped docker hub repo. If one doesn't exist, let's get with Ian and see what we can do. If that's not feasible, then leave as is.